blenderproc/python/utility/Utility.py

```diff
@@ -702,9 +702,9 @@ def is_any_active() -> bool:
 class NumpyEncoder(json.JSONEncoder):
     """ A json encoder that is also capable of serializing numpy arrays """
 
-    def default(self, obj: Any):
+    def default(self, o: Any):
         # If its a numpy array
-        if isinstance(obj, np.ndarray):
+        if isinstance(o, np.ndarray):
             # Convert it to a list
-            return obj.tolist()
-        return json.JSONEncoder.default(self, obj)
+            return o.tolist()
+        return json.JSONEncoder.default(self, o)
```

Comment:
This was undone in the other PR, is there a merge missing?

Reply:
No, this is correct. You just made me change it :D 

This is a derived function and the original parameter is called `o`, renaming it is a bad style.

Reply:
Ah okay :D So its the fault of the JSONEncoder authors then :D

Reply:
Exactly :D